pkg/snapshot/snapshotter/garbagecollector.go

```diff
@@ -193,24 +192,28 @@ func getSnapStreamIndexList(snapList brtypes.SnapList) []int {
 	return snapStreamIndexList
 }
 
-// garbageCollectChunks deletes the chunks in the store from snaplist starting at index low (inclusive) till high (exclusive).
-func garbageCollectChunks(store brtypes.SnapStore, snapList brtypes.SnapList, low, high int) {
-	for index := low; index < high; index++ {
+// garbageCollectChunks deletes the chunks in the store from snaplist.
+func garbageCollectChunks(store brtypes.SnapStore, snapList brtypes.SnapList) brtypes.SnapList {
+	var snapListWithoutChunks brtypes.SnapList
+	for index := 0; index < len(snapList); index++ {
```

Comment:
```suggestion
	for _, snap := range snapList {
```
Can we use `range`instead of `index`